armbalancer.go

```diff
@@ -82,8 +85,17 @@ type recyclableTransport struct {
 func newRecyclableTransport(id int, parent *http.Transport, host string, recycleThreshold, minReqsBeforeRecycle int64) *recyclableTransport {
 	tx := parent.Clone()
 	tx.MaxConnsPerHost = 1
+	hostUrl, err := url.Parse("https://" + host)
```

Comment:
`newRecyclableTransport` is invoked for each pooled transport - let's move the URL parsing up the stack to `New`.

Reply:
fixed in new commit